src/sdk.ts

```diff
@@ -18,9 +18,9 @@ import { NATIVE } from './wrapper';
  * @param options Options for the SDK
  * @param originalInit The init function of the sibling SDK, leave blank to initialize with `@sentry/browser`
  */
-export function init<O extends BrowserOptions>(
-  passedOptions: CapacitorOptions & O,
-  originalInit: (passedOptions: O) => void = browserInit,
+export function init<T>(
+  passedOptions: CapacitorOptions & T,
+  originalInit: (passedOptions: T & BrowserOptions) => void = browserInit,
```

Comment:
For react and angular nothing bad was happening since they just use BrowserOptions, but for Vue, they use VueOptions, extending BrowserOptions with some additional parameters. The way it was implemented didn't allow users to refer to the VueOptions on their code and it's required for initialising the vue SDK

Reply:
The sampling build tests will also assure  that there isn't a break change since they use parameters from SentryBrowser, SentryCapacitor and SentryVue